src/ResourceManager/MachineLearningCompute/Commands.MachineLearningCompute/Cmdlets/TestAzureRmMlOpClusterSystemServicesUpdateAvailability.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Microsoft.Azure.Commands.MachineLearningCompute.Models;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.Azure.Management.MachineLearningCompute;
+using Microsoft.Azure.Management.MachineLearningCompute.Models;
+using Microsoft.Rest.Azure;
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.MachineLearningCompute.Cmdlets
+{
+    [Cmdlet(VerbsDiagnostic.Test, CmdletSuffix + "SystemServicesUpdateAvailability")]
+    [OutputType(typeof(CheckSystemServicesUpdatesAvailableResponse))]
```

Comment:
@shutchings same comment about wrapping this in a PS class

Reply:
Done.